### ISSUE TITLE
Javascript: Does not recognise class members declared with decorators

### ISSUE THREAD

**Issue by ravisumit33** [universal-ctags/ctags#2562]
Name of the parser - Javascript

The definitions fail to recognise members declared with decorators (such as MobX's [@observable](https://mobx.js.org/refguide/observable-decorator.html) and [@computed](https://mobx.js.org/refguide/computed-decorator.html)).

Class methods written after methods having decorators are also not recognised.

*****

I have not specied any local configurations. Just used following command

```
$ ctags -R .
```

The version of ctags:

```
$ ctags --version
Universal Ctags 0.0.0(1c004b3), Copyright (C) 2015 Universal Ctags Team
Universal Ctags is derived from Exuberant Ctags.
Exuberant Ctags 5.8, Copyright (C) 1996-2009 Darren Hiebert
  Compiled: May 20 2020, 01:41:28
  URL: https://ctags.io/
  Optional compiled features: +wildcards, +regex, +iconv, +option-directory, +xpath, +case-insensitive-filenames, +packcc
```

How do you get ctags binary:

 macosx binary taken from Universal-ctags/homebrew-universal-ctags project

**Comment by masatake** [comment]
Thank you for reporting.

I don't know JavaScript well. As I wrote in the template, an example input and the expected output are needed. In addition, links for language (or something) references for "decorators" may help us.

**Comment by ravisumit33** [comment]
Following is link to language reference
[decorators](https://github.com/tc39/proposal-decorators)

decorators can be used with classes, properties as well as methods. You can find several such examples in the link above.

**Comment by ravisumit33** [comment]
h[ttps://github.com/universal-ctags/ctags/issues/2154](2154) is similar

**Comment by jafl** [comment]
@masatake I believe this ticket can be closed, because you fixed it in #2159

**Comment by masatake** [comment]
@jafl, thank you.

@ravisumit33, if your issue is still reproduced in the latest code taken from our git repo, please, reopen this.
